saleor/seo/models.py

```diff
@@ -23,6 +23,7 @@ class SeoModelTranslation(Translation):
     seo_description = models.CharField(
         max_length=300, blank=True, null=True, validators=[MaxLengthValidator(300)]
     )
+    slug = models.SlugField(max_length=255, allow_unicode=True, null=True)
```

Comment:
Not sure if this is proper place for adding `slug`. `SeoModelTranslation` is a representation of `SeoModel`, which where a model can inherit from it. 
In case of adding any new model (or adding new translation to existing model), this will automatically add `slug` field. 
```python

class NewTranslationModel(SeoModelTranslation):
    name = models.CharField(max_length=250, default="")
    # migration will add slug field
    
class NewTranslatableModel(SeoModel):
    name = models.CharField(max_length=250, unique=True)
    # doesn't have slug field
```

IMHO `slug` should be added explicitly to the translation models (or by a new abstract class).

Reply:
I see your argument, but I am not sure there can be a use-case where you would create new model with meta description and title were you wouldn't want to translate slug as well. The main reason why I want to translate them is to improve seo. If this is your decision, I will create a new class, but I feel like this separation is unnecessarily adding complexity to codebase (little complexity, but still)

Reply:
Let's get second opinion in that case :) 
@fowczarek  @maarcingebala  What do you think?

Reply:
> IMHO slug should be added explicitly to the translation models (or by a new abstract class).

I also agree with @korycins.



@pawelromanskidajar Huge kudos for amazing PR :)